cluster/pom.xml

```diff
@@ -34,11 +34,6 @@
         <cluster.ut.skip>${cluster.test.skip}</cluster.ut.skip>
     </properties>
     <dependencies>
-        <dependency>
-            <groupId>org.apache.thrift</groupId>
-            <artifactId>libthrift</artifactId>
```

Comment:
I think we can only remove the version here.
In the parent pom, the libthrift is claimed in dependencyManagement, rather than dependencies.

Reply:
@jixuan1989 the `cluster` module is dependent on `service-rpc`, and `service-rpc` module has the `libthrift` dependency which version extends from parent. So we don't need to add the `libthrift` dependency to cluster module separately.

```
service-rpc/pom.xml

 <dependency>
            <groupId>org.apache.thrift</groupId>
            <artifactId>libthrift</artifactId>
</dependency>
```